src/os/bluestore/BlueStore.cc

```diff
@@ -18299,7 +18324,14 @@ void BlueStore::_log_alerts(osd_alert_list_t& alerts)
   } else if (!spillover_alert.empty()){
     spillover_alert.clear();
   }
-
+  _trim_slow_op_event_queue(mono_clock::now());
```

Comment:
Why not returning amount of remaining entries in slow_op_event_queue as you did in BlockDevice? Uniformity is good. ;)